benchmark/manage_gatling_client.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+set -e
+set -o pipefail
+# set -x
+#Functions
+function usage {
+  echo "Setup of test clients for the benchmark test with a pre-defined secret"
+  echo "Usage: $(basename $0) -k KEYCLOAK_HOME -r REALM_NAME -c CLIENT_ID -s SERVER_URL [-d]" 2>&1
+  echo '-k keycloak home path ex: /home/opt/keycloak-18.0.0 : default value KEYCLOAK_HOME env variable'
+  echo '-r realm : default value realm-0'
+  echo '-c service account enabled client name : default value gatling'
+  echo '-s keycloak host server url : default value http://localhost:8081/auth'
+  echo '-d delete the client and realm before re-creating them: default value is false'
+  exit 1
+}
+
+function install_kcb {
+  echo "Installing kcadm.sh"
```

Comment:
This prompt on the console "Installing..." would indicate something would be permanently installed and would be in effect after the script has been run. As it is setting just the path, please rename the method and the prompt.